config/routes.rb

```diff
@@ -7,30 +7,28 @@
           :defaults => { apiv: 'v2' },
           :apiv => /v2/,
           :constraints => ApiConstraints.new(version: 2, default: true) do
-      constraints(:id => %r{[^\/]+}) do
```

Comment:
Was this never needed? I can't find anything in the Rails upgrade guide relating to this.